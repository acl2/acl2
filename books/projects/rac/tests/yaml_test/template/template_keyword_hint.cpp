#include <ac_int.h>

// RAC begin

template <int n>
int foo(ac_int<64, false> x) {
  return x.slc<n>(0);
}
