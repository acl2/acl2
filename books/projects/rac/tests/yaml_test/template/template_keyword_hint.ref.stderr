template_keyword_hint.cpp:7 (15-16): syntax error, unexpected ID, expecting NAT
  7 |   return x.slc<n>(0);
    |                ^
note: Consider adding the `template` keyword: ID.template slc<...
